pci.ids

```diff
@@ -9,6 +9,146 @@
 #	device  device_name				<-- single tab
 #		subvendor subdevice  subsystem_name	<-- two tabs
 
+
+#ETH Cards
+#---------
+
+1000  Broadcom / LSI
+	16d7  BCM57414 NetXtreme-E 10Gb/25Gb RDMA Ethernet Controller
+		14e4 1402  BCM957414A4142CC 10Gb/25Gb Ethernet PCIe
+
+<<<<<<< HEAD
+=======
```

Comment:
This is a git conflict marker. You should have resolved and removed that before doing `git add`, committing and pushing.